circuits/cpp/barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/goblin_translator_circuit_builder.cpp

```diff
@@ -0,0 +1,275 @@
+/**
+ * @file goblin_translator_circuit_builder.cpp
```

Comment:
I simply moved this code from .test.cpp and commented out all info and variables used just for info (messed with fuzzing)